pkg/ebpf/tracee.go

```diff
@@ -1404,6 +1404,10 @@ func (t *Tracee) Run(ctx gocontext.Context) error {
 	t.invokeInitEvents()
 	t.triggerSyscallsIntegrityCheck(trace.Event{})
 	t.triggerSeqOpsIntegrityCheck(trace.Event{})
+	err := t.triggerMemDump()
+	if err != nil {
+		return err
```

Comment:
Should this fail the whole run?

Reply:
yes as this means that the symbol provided doesn't exist for instance.

Reply:
```
	t.invokeInitEvents()
	t.triggerSyscallsIntegrityCheck(trace.Event{})
	t.triggerSeqOpsIntegrityCheck(trace.Event{})
```
All these don't fail the whole run. Why the inconsistency?
Since this event is triggered once (and not attached to any kprobe/tracepoint) - I think logging an error would be enough.

Reply:
So I fixed it for now - warning if there was an issue.
In the future when we implement argument reception from the user we will fail if the one of the arguments wasn't what we expected (for instance - symbol doesn't exists)
This is the issue: #2559